tests/queries/0_stateless/02961_drop_tables.reference

```diff
@@ -0,0 +1,3 @@
+02961_tb1
```

Comment:
Someone looking at the results file will have no idea which of the statements in the .sql test created them.

Please add "separators" a la `SELECT '-- check which tables exist in db1'` to the SQL test.